lib/slots.ts

```diff
@@ -17,37 +19,41 @@ export type GetSlots = {
 
 const getMinuteOffset = (date: Dayjs, step: number) => {
   // Diffs the current time with the given date and iff same day; (handled by 1440) - return difference; otherwise 0
-  const minuteOffset = Math.min(date.diff(dayjs().startOf("day"), "minutes"), 1440) % 1440;
+  const minuteOffset = Math.min(date.diff(dayjs().startOf("day"), "minute"), 1440) % 1440;
```

Comment:
Singular as this is documented, plural is undocumented; you never know when there is a breaking change to remove support for pluralization.